frontend/packages/ux-editor/src/components/Properties/DataModelBindings.tsx

```diff
@@ -39,18 +38,14 @@ export const DataModelBindings = (): React.JSX.Element => {
     setMultipleAttachments(updatedValue);
     const updatedComponent = {
       ...formItem,
+      itemType: 'COMPONENT' as const,
```

Comment:
```suggestion
      itemType: 'COMPONENT',
```

Nitpick, but I don't think you need to specifiy the type. Here is from the type definition:
<img width="1017" height="272" alt="image" src="https://github.com/user-attachments/assets/74081f17-55a4-46e0-b085-4ba9192ce90e" />